### PR TITLE
Pushbullet Webhook notification

### PR DESCRIPTION
**Pull request by cirow**
Implemented webhook notifications to https://www.pushbullet.com/
This is based mostly on the pushover.sh code

Pushbullet API Docs: https://docs.pushbullet.com/#create-chat

### CODE REVIEW

File: notify/pushbullet.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  PUSHBULLET_TOKEN="${PUSHBULLET_TOKEN:-$(_readaccountconf_mutable PUSHBULLET_TOKEN)}"
+  if [ -z "$PUSHBULLET_TOKEN" ]; then
+    PUSHBULLET_TOKEN=""
+    _err "You didn't specify a Pushbullet application token yet."
+    return 1
+  fi
+  _saveaccountconf_mutable PUSHBULLET_TOKEN "$PUSHBULLET_TOKEN"
+
+  PUSHBULLET_DEVICE="${PUSHBULLET_DEVICE:-$(_readaccountconf_mutable PUSHBULLET_DEVICE)}"
+  if [ -z "$PUSHBULLET_DEVICE" ]; then
```
Comment (Neilpang) [inline notify/pushbullet.sh:23]: Is this a mistake?

**Comment by Neilpang** [comment]
please also add usage here: https://github.com/acmesh-official/acme.sh/wiki/notify

File: notify/pushbullet.sh (hunk 1 above)
Comment (cirow) [reply notify/pushbullet.sh:23]: It is! Sorry for that. It should be alright now

**Comment by cirow** [comment]
Documentation added to https://github.com/acmesh-official/acme.sh/pull/3599#issuecomment-881495943

File: notify/pushbullet.sh (hunk 1 above)
Comment (Neilpang) [reply notify/pushbullet.sh:23]:
```
if [ -z "$PUSHBULLET_DEVICE" ]; then
  _clearaccountconf_mutable PUSHBULLET_DEVICE
else
  _saveaccountconf_mutable PUSHBULLET_DEVICE "$PUSHBULLET_DEVICE"
fi
```

**Pull request merged**